content/about.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "About"
+date: 2017-07-03T14:41:50-04:00
+draft: false
+mainpage: false
+projectpage: false
+projectslug: ""
+projectname: ""
+listdescription: ""
+---
+
+### About
+
+WorldWind is a free, open source API for a virtual globe.  Written in Java so as to be cross-platform and easy-to-use, WorldWind allows developers to quickly and easily create interactive visualizations of 3D globe, map and geographical information.
+
+<img align="left" src="/img/800px-worldwind_screen.jpg" width="30%" hspace="10" vspace="10"> WorldWindOrganizations across the world use WorldWind to monitor weather patterns, visualize cities and terrain, track the movement of planes, vehicles and ships, analyze geospatial data, and educate people about the Earth.
+
+<img align="right" src="/img/wwj_applet_13_250.jpg" width="30%" hspace="10" vspace="10">Because WorldWind is completely open source, extending the API is simple and easy to do, creating a powerful platform for giving any application the means to express, manipulate and analyze spatial data.  WorldWind technology can be incorporated into Windows, Mac and Linux applications, web pages and web applications, and soon, mobile applications as well.
+
+Curated by NASA, work on WorldWind began in 2002, released under the NASA Open Source Agreement (NOSA).  The project was ported to Java in 2006, and in November 2009 was awarded NASA Software of the Year.  Since then it has developed a global community of users, from government, education, the non-profit sector, as well as private industry.
+
+WorldWind is different from a 3D globe like Google Earth because it is not a completed application targeted at end users.  Instead, it is an SDK (software development kit) that software engineers can use to build their own applications.  WorldWind provides a geographic rendering engine for powering a wide range of projects, from satellite tracking systems to flight simulators.
```

Comment:
Minor suggestion to remove negative context: Change "WorldWind is different from a 3D globe like Google Earth because it is not a completed application targeted at end users." to "WorldWind is different from a 3D globe like Google Earth because it is not an application targeted at specific end users."